reference/micropython/network/WLAN.pyi

```diff
@@ -19,17 +19,40 @@ class WLAN:
     """
 
     PM_NONE: int
-    PM_POWERSAVE: int
     PM_PERFORMANCE: int
+    PM_POWERSAVE: int
 
     """\
     Allowed values for the ``WLAN.config(pm=...)`` network interface parameter:
     
-    * ``PM_PERFORMANCE``: enable WiFi power management to balance power
-    savings and WiFi performance
-    * ``PM_POWERSAVE``: enable WiFi power management with additional power
-    savings and reduced WiFi performance
-    * ``PM_NONE``: disable wifi power management
+    * ``PM_NONE``: disable WiFi power management (always on, highest performance, highest power consumption)
+    * ``PM_PERFORMANCE``: enable WiFi power management to balance power savings and WiFi performance
+    * ``PM_POWERSAVE``: enable WiFi power management with additional power savings and reduced WiFi performance
+    
+    **Note:** The numeric values of these constants are platform-specific:
+    
```

Comment:
Please summarize the different platform values in a table rather than extensive text